src/main/java/org/jabref/logic/importer/util/JSONEntryParser.java

```diff
@@ -229,4 +231,67 @@ public static BibEntry parseSpringerJSONtoBibtex(JSONObject springerJsonEntry) {
 
         return entry;
     }
+
+    /**
+     * @implNote <a href="https://developer.ieee.org/docs/read/Metadata_API_responses">documentation</a>
+     */
+    public static BibEntry parseIEEEJSONtoBibtex(JSONObject jsonEntry, Character keywordSeparator) {
```

Comment:
For me this belongs to the IEEE class.

Reply:
Good point. Moved it to the fetcher (as well as the other parsing code in that class)